websites/C/CHZZK/presence.ts

```diff
@@ -0,0 +1,112 @@
+const presence = new Presence({
+		clientId: "1232944311415603281",
+	}),
+	getStrings = async () => {
+		return presence.getStrings(
+			{
+				play: "general.playing",
+				pause: "general.paused",
+				live: "general.live",
+				browse: "general.browsing",
+				ad: "youtube.ad",
+				watchingLive: "general.watchingLive",
+				watchingVid: "general.watchingVid",
+				watchStream: "general.buttonWatchStream",
+				watchVideo: "general.buttonWatchVideo",
+			},
+			oldLang
+		);
+	},
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://cdn.discordapp.com/app-assets/1232944311415603281/1232944461047267338.png?size=512",
+}
+
+const enum ChzzkAssets {
+	Browse = "https://cdn.discordapp.com/app-assets/1232944311415603281/1232952455562526731.png",
+	Live = "https://cdn.discordapp.com/app-assets/1232944311415603281/1232952455369850910.png",
+	Play = "https://cdn.discordapp.com/app-assets/1232944311415603281/1232952455625707581.png",
+	Pause = "https://cdn.discordapp.com/app-assets/1232944311415603281/1232952455101419540.png",
```

Comment:
Assets should be 512x512px. You can also consider using the default assets.
```suggestion
	Browse = "https://cdn.discordapp.com/app-assets/1232944311415603281/1232952455562526731.png?size=512",
	Live = "https://cdn.discordapp.com/app-assets/1232944311415603281/1232952455369850910.png?size=512",
	Play = "https://cdn.discordapp.com/app-assets/1232944311415603281/1232952455625707581.png?size=512",
	Pause = "https://cdn.discordapp.com/app-assets/1232944311415603281/1232952455101419540.png?size=512",
```